### PR TITLE
qrcp 0.11.3

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>## Changelog
* a98e754 fix(config): correct https-related fields in wizard (#334)
* 3b803b1 fix(payload): Add UTF-8 filename* in Content-Disposition (#332)
* 0a0bd60 fix(server): Fix link when external IP is IPv6 (#335)

## Downloads
| Platform | Download link                                                                                                                 |
|----------|-------------------------------------------------------------------------------------------------------------------------------|
| Linux    | [deb package](https://github.com/claudiodangelis/qrcp/releases/download/0.11.3/qrcp_0.11.3_linux_x86_64.deb)          |
| Linux    | [RPM package](https://github.com/claudiodangelis/qrcp/releases/download/0.11.3/qrcp_0.11.3_linux_x86_64.rpm)          |
| macOS    | [macOS package](https://github.com/claudiodangelis/qrcp/releases/download/0.11.3/qrcp_0.11.3_macOS_x86_64.tar.gz)     |
| Windows  | [Windows package](https://github.com/claudiodangelis/qrcp/releases/download/0.11.3/qrcp_0.11.3_Windows_x86_64.tar.gz) |

Refer to the list of assets below for all supported platform.

</pre>
</details>

### CODE REVIEW

File: .github/autobump.txt
Comment (carlocab) [inline .github/autobump.txt:1]: Let's not change unrelated lines please

Comment (chenrui333) [reply .github/autobump.txt:1]: I did the sort as I always did before, so the lines got changed when you made the change? how can we ensure the sorting then?

**Review by branchv — APPROVED** [review]
